android_jni/avifandroidjni/src/main/java/org/aomedia/avif/android/AvifDecoder.java

```diff
@@ -110,11 +110,13 @@ public static boolean decode(ByteBuffer encoded, int length, Bitmap bitmap) {
    *     cores as the thread count. Negative values are invalid. When this value is > 0, it is
    *     simply mapped to the maxThreads parameter in libavif. For more details, see the
    *     documentation for maxThreads variable in avif.h.
+   * @param isScale Is enable image scale in decoding. If true the encoded AVIF image size while be
+   *                decode with size of bitmap.
    * @return true on success and false on failure. A few possible reasons for failure are: 1) Input
    *     was not valid AVIF. 2) Bitmap was not large enough to store the decoded image. 3) Negative
    *     value was passed for the threads parameter.
    */
-  public static native boolean decode(ByteBuffer encoded, int length, Bitmap bitmap, int threads);
+  public static native boolean decode(ByteBuffer encoded, int length, Bitmap bitmap, int threads, boolean isScale);
```

Comment:
couple of points:
1) This is an API breaking change. So it should be done as a new overloaded function with false being passed from the existing ones to keep backwards compatibility.

2) i don't think the `isScale` parameter is necessary at all. we could just update the function comments to say that if the `bitmap` dimensions do not match the decoded image's dimensions, then the decoded image will be scaled to match the bitmap's dimensions (i.e.) scaling could be the default behavior.

Reply:
For the first point, I will modify it
For the second point, what I start to think is not affecting the original API. If you join this feature uniformly, I think it can be removed